Scripts/Audio.cs

```diff
@@ -183,4 +183,138 @@ public string output
     }
 
 
+
+    [Serializable]
+    public class SpeechRecognitionOptions
+    {
+        public enum SpeechRecognitionService
+        {
+            Unified,
+            Google,
+            Aws,
+            Deepgram
+        }
+
+        public enum SpeechRecognitionLanguageCode
```

Comment:
I think we should not be prescriptive with the language code and just accept any string directly. Different services allow different language codes and may also add or change accepted codes in the future